docs/actions/require-verified-address.mdx

```diff
@@ -1,47 +1,203 @@
 ---
 id: require-verified-address
-title: Require verified address
+title: Address verification
```

Comment:
This document now covers everything about account verification actions. In my view it's quite hidden under "Guides->Ory Actions->Identity Management->Address Verification" if I compare this with other docs where it would be a first-level navigation item under guides, but I just want to get the content right for now. Restructuring is then the job of the person responsible for docs.